wordsmithed some verbage on FAQ

emklein opened the pull request.

emklein committed:
wordsmithed some verbage on FAQ

ARoski commented on become_active.md:
```diff
@@ -25,15 +25,15 @@ In joining Triangle, you will become part of a brotherhood of over 29,000 men. T
 Q: What is the minimum GPA requirement?
 
-A: The minimum GPA requirement is 2.7. If you do not meet this GPA, we will be more than willing to help you out in your school work so that you may increase it.
+A: The minimum GPA requirement is 2.7. If you do not meet this GPA, don't sweat it. With one of the highest Greek GPA on at Michigan Tech, we will be more than willing to help you with your school work so that you may increase it. Our brothers thrive on helping individuals grow and supporting others in their success.
```
There is a grammar mistake here saying "...highest Greek GPA on at Michigan Tech". Also the last sentence needs to be reworded, it doesn't sound quite right. Maybe change it to "Our brothers strive to help individuals grow as well as support others in their success."

emklein committed:
Merge branch 'master' into patch-1

emklein committed:
fixed gpa FAQ typo to proper english

"highest greek GPAs at Michigan Tech" and "help individuals grow as well as support others in their success"

raveious committed:
Merge branch 'master' into patch-1

emklein committed:
changed thrive to strive in GPA FAQ
